Pipfile

```diff
@@ -12,4 +12,4 @@ bs4 = "*"
 pylint = "*"
 
 [requires]
-python_version = "3.6"
```

Comment:
Keeping a fixed Python version is better - whichever latest version works correctly.